lms/templates/verify_student/make_payment_step.underscore

```diff
@@ -98,11 +98,10 @@
     <% } %>
 
   <% if ( isActive ) { %>
-  <div class="nav-wizard is-ready center">
+  <div class="payment-buttons nav-wizard is-ready center">
     <input type="hidden" name="contribution" value="<%- minPrice %>" />
-    <a class="next action-primary" id="pay_button" tab-index="0">
-      <%- gettext( "Continue to payment" ) %> ($<%- minPrice %>)
-    </a>
+    <span>price: $<%- minPrice %></span>
+    <!-- payment buttons will go here -->
```

Comment:
Pretty sure this comment will show up in the output HTML.  Would it be possible to use JS comments instead?
```
<%
// payment buttons will go here
%>
```